src/write.ts

```diff
@@ -117,7 +117,58 @@ function getCurrentRepo() {
 }
 
 function floatStr(n: number) {
-    return Number.isInteger(n) ? n.toFixed(1) : n.toString();
+    return Number.isInteger(n) ? n.toFixed(0) : n.toFixed(2);
```

Comment:
Why is this changed? I think the `n.toFixed(2)` is not acceptable. For example, when the value is much smaller than zero, the value is truncated to zero.

```
> 0.00001.toFixed(2)
'0.00'
```

Please keep `toString()` here.

Reply:
We are using this function only for thresholds which are never near to `0`. Having it `toString()` printed out more than 10 character string which I said before doesn't make the comment look nice because too much width on the table.

Reply:
This action is not only for your case. Unless you ensure that all benchmark tools' outputs, please use `toString` conservatively.

Reply:
Can I add an option for this then?

Reply:
How about make the logic cleverer to fit your case?

For example:

```javascript
if (Number.isInteger(n)) {
    return n.toString();
}

if (n > 0) {
    return n.toFixed(2);
}

return n.toString(); // Fallback
```

Here when `n` is larger than zero, it means that it's big enough and we can truncate the number. So we can use `toFixed(2)`. Otherwise we faillback into  `toString`.

Introducing new option also introduces more complexity. So I'm trying not to add new option as much as possible.

BTW, what is the 'ugly' case you're facing (just curious)?

Reply:
You mean `n > 1` in the if loop condition, right?

Check https://github.com/clap-rs/clap/pull/1724#commitcomment-37649200. That much info is not needed for the ratio.

So, I updated the PR with your proposed change.

Reply:
Yes, you're correct. Thanks, I'll check it.

Reply:
So, I realized the condition should probably be `n > 0.1`, not `n > 1`. Because if n is 0.98499484849, then the whole thing will be printed.

Reply:
ah, I see. I'll fix up before release